src/app/form-password/form-password.component.ts

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable no-unused-expressions */
+import {Component, OnInit, forwardRef} from '@angular/core';
+import {
+  ControlValueAccessor,
+  Validators,
+  FormGroup,
+  FormControl,
+  NG_VALUE_ACCESSOR,
+} from '@angular/forms';
+
+@Component({
+  selector: 'app-form-password',
+  templateUrl: 'form-password.component.html',
+  styleUrls: ['form-password.component.scss'],
+  providers: [
+    {
+      provide: NG_VALUE_ACCESSOR,
+      useExisting: forwardRef(() => FormPasswordComponent),
+      multi: true,
+    },
+  ],
+})
+export class FormPasswordComponent implements OnInit, ControlValueAccessor {
+  public pwdForm: FormGroup = new FormGroup({
+    // eslint-disable-next-line @typescript-eslint/unbound-method
+    pwd: new FormControl('', [Validators.required]),
+  });
+
+  hide = true;
+
+  // constructor() {}
+
+  ngOnInit(): void {}
+
+  public onTouched: () => void = () => {};
+
+  writeValue(val: any): void {
+    val && this.pwdForm.setValue(val, {emitEvent: false});
```

Comment:
questa forma qui è breve ma poco espressiva e eslint la segna come errore; sarebbe meglio scrivere if (val) {this....}